ChangeLog

```diff
@@ -1,5 +1,18 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x released xxxx-xx-xx
+
+Changes
+   * Added config.h option MBEDTLS_NO_UDBL_DIVISION, to prevent the use of
+     64-bit division.
+   * Added config.h option MBEDTLS_TYPE_UDBL to allow configuring the
```

Comment:
@gilles-peskine-arm Yes I understand, but for the purpose of documentation I think it's clearer to have an option to generally disable the use double-width integers in the bignum code, regardless of whether the reason is that (1) there's no such type or (2) there is such a type but it lacks division.